### DISCUSSION TITLE
How delete bounding box?

### DISCUSSION THREAD

**Discussion by yarnik1** [ImageMagick/ImageMagick#2789] [Help]
I'm trying to minify the file and clear all the metadata, but these actions I see several fields that are not erased:
`convert $file -sampling-factor 2x2,1x1,1x1 -strip -quality 90 -interlace JPEG -colorspace ycbcr -resize 800x600 $dest`;
`identify -verbose $dest`;

```
  Convex hull: 0,0 714,0 714,591 713,599 0,599 0,0 
  Minimum bounding box: 714,0 714,599 0,599 0,0 
  Properties:
    date:create: 2020-11-03T11:42:10+00:00
    date:modify: 2020-11-03T11:43:45+00:00
    jpeg:colorspace: 2
    jpeg:sampling-factor: 2x2,1x1,1x1
    minimum-bounding-box:_p: 0,0
    minimum-bounding-box:_q: 714,0
    minimum-bounding-box:_v: 713,599
    minimum-bounding-box:angle: 0
    minimum-bounding-box:area: 427686
    minimum-bounding-box:height: 714
    minimum-bounding-box:unrotate: -0
    minimum-bounding-box:width: 599
    signature: b150bfeb9af5c22d308fc0aa44890aa24f20f53b2f620b00fb4c40ece9ee9e1d
  Artifacts:
    verbose: true
```

**Comment by snibgo** [comment]
You can use "+set minimum-bounding-box:angle" etc.

**Comment by yarnik1** [comment]
Can I delete it completely?

**Reply by fmw42** [reply]
No, not at this time. Those are Imagemagick computed properties. +set does not work nor does -strip. An enhancement to remove those would be needed.

**Comment by urban-warrior** [comment]
Identify describes the format and characteristics of one or more image files.  Some of the characteristics are computed as the image is identified such as the minimum bounding box.  As such there is no method to suppress these properties.  You can however, suppress other properties with this command:

`identify -verbose -strip +set jpeg:colorspace +set jpeg:sampling-factor $dest`

We could save the minimum bounding box attributes as artifacts (ephemeral) rather than properties (persistent).  However, that likely does not help you.  Also keep in mind, you can save image characteristics as JSON and use a query to only extract key value pair that you are interested in.
